roles/development/files/bin/make-changelog.py

```diff
@@ -105,24 +150,50 @@ def main(username, project, version):
 
         relstr = "Merge branch 'release"
         pullstr = "Merge pull request #"
-
-        commits = [(slug, msg) for slug, msg in commits if relstr not in msg]
-        pulls = [(slug, msg) for slug, msg in commits if pullstr in msg]
-        commits = [(slug, msg) for slug, msg in commits if pullstr not in msg]
+        get_pull_info = get_pull_info_github
+        pull_url = pull_url_for_github
+        commit_url = commit_url_for_github
+
+        if pagure:
+            pullstr = "Merge #"
+            get_pull_info = get_pull_info_pagure
+            pull_url = pull_url_for_pagure
+            commit_url = commit_url_for_pagure
+
+        commits = [
+            (slug, comdate, msg)
+            for slug, comdate, msg in commits
+            if relstr not in msg]
+        pulls = [
```

Comment:
So does this line produce an *empty* list of ``pulls`` for pagure always?  If so, some comment should be added indicating why that that is the case.

It is confusing that we build ``pulls =`` here .. but then for pagure we *add more pulls* down below with ``pulls.extend(get_pull_requests_pagure(...))``.